### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by CostircoNazar**

### CODE REVIEW

File: src/form/Form.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, {useState} from 'react';
-import Input from "../inputComponent/Input";
-import SubmitButton from "../buttonComponent/SubmitButton";
+import Input from "../input/Input";
+import SubmitButton from "../button/SubmitButton";
 import './Foms.css'
```
Comment (HrudakovSerhii) [inline src/form/Form.js]: Please change component forder name from button to SubmitButton.

Same changes for Input. So "import Input from "../input/Input";" -> import Input from "../Input/Input";

File: src/container/Styles.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .containerForm > Form {
```
Comment (HrudakovSerhii) [inline src/container/Styles.css]: 1. Change containerForm to formContainer. We using camelCase notation and first word related with component name and then component usage description. In this case formContainer = component name + component description. Of course if it makes sense. Let's say for SubmitButton -> description name + TYPE of component. 
2. Change this syntax to .formContainer { form { ...form styles } }

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
-import Container from "./containerForm/Container";
+import {Container} from "./container/Container";
```
Comment (HrudakovSerhii) [inline src/App.js]: This changes was only to show difference between export and default export. Please change export of the Container file to default

File: src/container/Container.js [hunk 4]
```diff
@@ -11,4 +11,4 @@ function Container() {
     )
```
Comment (HrudakovSerhii) [inline src/container/Container.js]: Here you can see inconsistence of names on component and it's styles. Your component name is Container but it's class name is containerForm. Please look to next comment below to see description.

**Review by HrudakovSerhii — COMMENTED** [review]
Please check comments. Make changes and let me know when you done

File: src/App.js (hunk 3 above)
Comment (CostircoNazar) [reply src/App.js]: Done

File: src/container/Container.js (hunk 4 above)
Comment (CostircoNazar) [reply src/container/Container.js]: Changed the div className also to 'formContainer'

File: src/SubmitButton/Styles.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+.defaultButton{
+    display: flex;
```
Comment (HrudakovSerhii) [inline src/SubmitButton/Styles.css]: Please change name of the class so it's consistent with name of the component (but start with small key)

File: src/container/Styles.css (hunk 2 above)
Comment (CostircoNazar) [reply src/container/Styles.css]: Done

File: src/App.js [hunk 6]
```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
-import Container from "./containerForm/Container";
+import FormContainer from "./formContainer/FormContainer";
```
Comment (HrudakovSerhii) [inline src/App.js]: Please rename component directory so it's consistent with component name (Capitalise first key)

File: src/SubmitButton/SubmitButton.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Styles.css'
+
+function SubmitButton(props) {
+    return(
+        <button className='defaultButton'>{props.label}</button>
```
Comment (HrudakovSerhii) [inline src/SubmitButton/SubmitButton.js]: Please change name of the class so it's consistent with name of the component (but start with small key)

File: src/formContainer/FormContainer.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+    return(
+        <div className='formContainer'>
+            <Form/>
+        </div>
+    )
+}
+
+export default FormContainer
+
```
Comment (HrudakovSerhii) [inline src/formContainer/FormContainer.js:15]: Please rename component directory so it's consistent with component name (Capitalise first key)

**Review by HrudakovSerhii — COMMENTED** [review]
Please read all comments below

File: src/App.js (hunk 6 above)
Comment (CostircoNazar) [reply src/App.js]: Corrected

File: src/SubmitButton/SubmitButton.js (hunk 7 above)
Comment (CostircoNazar) [reply src/SubmitButton/SubmitButton.js]: Corrected

File: src/formContainer/FormContainer.js (hunk 8 above)
Comment (CostircoNazar) [reply src/formContainer/FormContainer.js:15]: Corrected

File: src/App.js (hunk 6 above)
Comment (HrudakovSerhii) [reply src/App.js]: After folder renaming it should look like
import FormContainer from "./**FormContainer**/FormContainer";

Comment (CostircoNazar) [reply src/App.js]: It looks exactly like that
```suggestion
import FormContainer from "./FormContainer/FormContainer";
```

**Pull request merged**